encrypt-config/tests/resources/minifi.properties

```diff
@@ -24,6 +24,7 @@ nifi.provenance.repository.directory.default=${MINIFI_HOME}/provenance_repositor
 nifi.provenance.repository.max.storage.time=1 MIN
 nifi.provenance.repository.max.storage.size=1 MB
 nifi.flowfile.repository.directory.default=${MINIFI_HOME}/flowfile_repository
+nifi.flowfile.checkpoint.directory.default=${MINIFI_HOME}/flowfile_checkpoint
```

Comment:
I think we can omit this from test resources if it's the same as the default. It makes sense to add to the example config, but in test resources, it's just redundant.

Reply:
Updated in 28dfa20abfc159c5da28c85026fae4f459b430bb